packages/product-feed-heureka/composer.json

```diff
@@ -20,21 +20,43 @@
             "Tests\\ProductFeed\\HeurekaBundle\\": "tests/"
         }
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/molaux/PostgreSearchBundle.git"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/shopsys/doctrine2.git"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/shopsys/jparser.git"
+        }
+    ],
     "require": {
         "php": "^7.0",
         "ext-simplexml": "*",
+        "ext-pdo": "*",
+        "ext-json": "*",
+        "ext-libxml": "*",
+        "doctrine/collections": "^1.5",
+        "doctrine/dbal": "^2.7",
+        "doctrine/doctrine-bundle": "^1.8",
+        "doctrine/orm": "dev-doctrine-260-with-ddc1960-hotfix-and-ddc4005-hotfix as 2.6.0",
```

Comment:
Why do you require this particular commit and not @dev as with intaro/postgres-search-bundle and timwhitlock/jparser?

Reply:
Because  `@dev as 2.6.0` didn't work. I didn't investigate the problem any deeper.
(well I read about minimum stability and also branch aliasing, but it was never combined)